client/lib/user/store.js

```diff
@@ -15,3 +15,11 @@ export async function clearStore() {
 	store.clearAll();
 	await clearStorage();
 }
+
+export async function getUserId() {
+	return await store.get( 'wpcom_user_id' );
+}
+
+export async function setUserId( userId ) {
+	await store.set( 'wpcom_user_id', userId );
```

Comment:
This should also `return` the result.

I'd call the functions `getStoredUserId` and `setStoredUserId`, to make more clear what and where we are getting and setting.

Reply:
Good call. Done as part of 9cbfd723f8c1072acdac4450ea1ba3a49845cd2d